Update XHR property descriptions (#15578)
mdn/content: files/en-us/web/api/xmlhttprequest/index.md

- {{domxref("XMLHttpRequest.status")}} {{readonlyinline}}
  - : Returns the [HTTP response status code](/en-US/docs/Web/HTTP/Status) of the request.
- {{domxref("XMLHttpRequest.statusText")}} {{readonlyinline}}
  - : Returns a {{domxref("DOMString")}} containing the response string returned by the HTTP server. Unlike {{domxref("XMLHttpRequest.status")}}, this includes the entire text of the response message ("`OK`", for example).

    > **Note:** According to the HTTP/2 specification {{RFC(7540, "Response Pseudo-Header Fields", "8.1.2.4")}}, HTTP/2 does not define a way to carry the version or reason phrase that is included in an HTTP/1.1 status line.

- {{domxref("XMLHttpRequest.timeout")}}
  - : The time in milliseconds a request can take before automatically being terminated.
- {{domxref("XMLHttpRequest.upload")}} {{readonlyinline}}
  - : A {{domxref("XMLHttpRequestUpload")}} representing the upload process.
- {{domxref("XMLHttpRequest.withCredentials")}}

>   - : Whether or not cross-site `Access-Control` requests should be made using credentials such as cookies or authorization headers.

Reviewer: ```suggestion
  - : Returns `true` if cross-site `Access-Control` requests should be made using credentials such as cookies or authorization headers; otherwise `false`.
```
